test_scripts/unit_tests/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
+SHELL ["/bin/bash", "-c"]
+RUN apt-get update
+RUN apt-get install -y python procps git
+RUN git clone https://github.com/bats-core/bats-core.git /bats/
+RUN /bats/install.sh /usr/local
+RUN git clone https://github.com/ztombol/bats-support /usr/local/lib/bats-support/
+RUN git clone https://github.com/ztombol/bats-assert /usr/local/lib/bats-assert/
```

Comment:
Versioning again
If you can't use a release tarball then I'd stick all these clones together

Reply:
I should be able to use a release tarball